data/src/main/java/com/example/data/common/Connectivity.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.data.common
+
+interface Connectivity {
+    fun hasNetworkAccess(): Boolean
```

Comment:
Where is no network operation in this app

Reply:
Resolved